internal/rfc7512/rfc7512.go

```diff
@@ -0,0 +1,182 @@
+package rfc7512
+
+import (
+	"crypto"
+	"fmt"
+	"net/url"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/cbroglie/crypto11"
+)
+
+// A PKCS11URI represents a PKCS#11 object stored on a PKCS#11 token and
+// information for accessing it. The general form represented is:
+//
+//	pkcs11:path-attr[?query-attr]
```

Comment:
What does this mean? More examples of copious documentation of unfamiliar formats:
- https://golang.org/pkg/text/template/
- https://golang.org/src/time/time.go#L328
- https://golang.org/pkg/regexp/syntax/

Reply:
There already exists copious documentation of this unfamiliar format:
```
// For more information read: https://tools.ietf.org/html/rfc7512#section-2.3
```

Reply:
Try re-writing and summarizing the relevant parts in your own words.